src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -304,7 +304,9 @@ public IWebProxy Proxy
                 return _proxy;
             }
 
+#if NET46
```

Comment:
nit: I think it is unnecessary to put any if-def wrappers in this file.  This file is already only compiled for the NET46 build.